src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -284,11 +284,10 @@ def register_blob_source_uri_arguments(scope):
 
 register_cli_argument('storage account create', 'location', location_type, validator=get_default_location_from_resource_group)
 register_cli_argument('storage account create', 'account_type', help='The storage account type', **model_choice_list(ResourceType.MGMT_STORAGE, 'AccountType'))
-
 register_cli_argument('storage account create', 'account_name', account_name_type, options_list=('--name', '-n'), completer=None)
-
 register_cli_argument('storage account create', 'kind', help='Indicates the type of storage account.', default=enum_default(ResourceType.MGMT_STORAGE, 'Kind', 'storage'), **model_choice_list(ResourceType.MGMT_STORAGE, 'Kind'))
 register_cli_argument('storage account create', 'tags', tags_type)
+register_cli_argument('storage account create', 'https_only', help='Allows https traffic only to storage service.', action='store_true')
```

Comment:
Can this be changed after the fact (i.e. with update)?  If so, consider three_state_flag here.